index.html

```diff
@@ -2276,7 +2276,251 @@ <h1>Schedule Roster</h1>
+
+    <div class="ritz grid-container" dir="ltr">
```

Comment:
Not necessary, although nice to start with

Reply:
ok @mohitm15